CacheManager.StackExchange.Redis.SelfCleaning.Examples/ManualScenariosRunner.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Channels;
+using CacheManager.Core;
+using CacheManager.Redis;
+using CacheManager.StackExchange.Redis.SelfCleaning.Core;
+using CacheManager.StackExchange.Redis.SelfCleaning.Examples;
+using CacheManager.StackExchange.Redis.SelfCleaning.Examples.Scenarios;
+using StackExchange.Redis;
+using static CacheManager.StackExchange.Redis.SelfCleaning.Examples.Utilities;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning.Examples
+{
+    public class ManualScenariosRunner
+    {
+        private TimeSpan _cleanupInterval = TimeSpan.FromSeconds(0.1);
+        private TimeSpan _slidingExpiration = TimeSpan.FromSeconds(1);
+        private string _connectionString = "localhost:6379";
+
+        public void Run()
+        {
+            Console.WriteLine("Hello to you dear monsieur/mademoiselle, welcome to our Reidis-Mania! Lets run some weird scenarios!");
+            GetParametersFromUser();
+
+            RunConfiguredScenarios();
+            
+            Console.WriteLine("\n\n\n Done running scenarios, press any key to exit. See you next time!");
+            Console.Read();
+        }
+
+        private void RunConfiguredScenarios()
+        {
+            RunAnotherScenario(new SimpleSingleExpiredItemScenario(CreateCacheManager<int>, _slidingExpiration).RunScenario);
+            RunAnotherScenario(new SelfCleaningHermeticityScenario(CreateCacheManager<double>, 1, _slidingExpiration).RunScenario);
+            //RunAnotherScenario(new SelfCleaningHermeticityScenario(CreateCacheManager<double>, 5, _slidingExpiration).RunScenario);
+        }
+
+        private void RunAnotherScenario(Action scenario)
```

Comment:
1. The only reason is because I dont want to write the ********** again and again, so it wont look like that:
![image](https://user-images.githubusercontent.com/62211836/78679498-f705e980-78f2-11ea-9e5c-664eb719eacf.png)

If you think thats ok so I will inline it (and it will reduce the number of functions in the class)

anyway, fixed 2 + 3

Reply:
If it's just about looks, you can do something like:
```private static void PrintSeparationLine() => Console.WriteLine("\n\n\n ******************************* \n\n\n");```
your choice, tell me how'd you like it and I'll resolve

Reply:
So I prefer my old solution, with the RunSingleScenario method